tests/model_registry/conftest.py

```diff
@@ -177,16 +180,40 @@ def model_registry_instance(
 
 @pytest.fixture(scope="class")
 def model_registry_mysql_config(
-    model_registry_db_deployment: Deployment, model_registry_db_secret: Secret
+    request: FixtureRequest,
+    model_registry_db_deployment: Deployment,
+    model_registry_db_secret: Secret,
 ) -> dict[str, Any]:
-    return {
+    """
+    Fixture to build the MySQL config dictionary for Model Registry.
+    Expects request.param to be a dict. If 'sslRootCertificateConfigMap' is not present, it defaults to None.
+    If 'sslRootCertificateConfigMap' is present, it will be used to configure the MySQL connection.
+
+    Args:
+        request: The pytest request object
+        model_registry_db_deployment: The model registry db deployment
+        model_registry_db_secret: The model registry db secret
+    Returns:
+        dict[str, Any]: The MySQL config dictionary
+    """
+    param = request.param if hasattr(request, "param") else {}
+    config = {
         "host": f"{model_registry_db_deployment.name}.{model_registry_db_deployment.namespace}.svc.cluster.local",
         "database": model_registry_db_secret.string_data["database-name"],
         "passwordSecret": {"key": "database-password", "name": model_registry_db_deployment.name},
-        "port": 3306,
+        "port": param.get("port", 3306),
         "skipDBCreation": False,
         "username": model_registry_db_secret.string_data["database-user"],
     }
+    if param.get("sslRootCertificateConfigMap"):
+        ssl_config = param.get("sslRootCertificateConfigMap")
+        if ssl_config:
+            config["sslRootCertificateConfigMap"] = {
+                "name": ssl_config.get("name"),
```

Comment:
you can set the param in your test such that you can directly set `config["sslRootCertificateConfigMap"]=param.get("sslRootCertificateConfigMap")`. This way it would be None, if the param does not exist.